x-pack/legacy/plugins/canvas/public/apps/export/export/export_app.js

```diff
@@ -39,7 +40,7 @@ export class ExportApp extends React.PureComponent {
           </div>
           {Style.it(
             workpad.css,
-            <div className="canvasExport__stageContent">
+            <div className="canvasExport__stageContent" data-shared-items-count={pageElementCount}>
```

Comment:
This tells reporting how many elements it should expect on the page. Since the embeddables aren't loaded immediately sometimes, we need to tell reporting to wait and expect them to become available